recipe/install_libclang.sh

```diff
@@ -3,15 +3,3 @@ set -ex
 
 cd ${SRC_DIR}/clang/build
 make install
```

Comment:
??

Reply:
![image](https://github.com/user-attachments/assets/4cdeae70-22fc-474c-b279-c7064e99ac47)

I verified that all the package contains is the symlink to `libclang.so.13` so this is functioning as expected.